docs/source/routing/get-started.mdx

```diff
@@ -0,0 +1,293 @@
+  createReview(upc: ID!, id: ID!, body: String): Review
+    @join__field(graph: REVIEWS)
+}
+
+type Product
+  @join__type(graph: ACCOUNTS, key: "upc", extension: true)
+  @join__type(graph: INVENTORY, key: "upc")
+  @join__type(graph: PRODUCTS, key: "upc")
+  @join__type(graph: REVIEWS, key: "upc") {
+  upc: String!
+  weight: Int
+    @join__field(graph: INVENTORY, external: true)
+    @join__field(graph: PRODUCTS)
+  price: Int
+    @join__field(graph: INVENTORY, external: true)
+    @join__field(graph: PRODUCTS)
+  inStock: Boolean @join__field(graph: INVENTORY)
+  shippingEstimate: Int @join__field(graph: INVENTORY, requires: "price weight")
+  name: String @join__field(graph: PRODUCTS)
+  reviews: [Review] @join__field(graph: REVIEWS)
+  reviewsForAuthor(authorID: ID!): [Review] @join__field(graph: REVIEWS)
+}
+
+type Query
+  @join__type(graph: ACCOUNTS)
+  @join__type(graph: INVENTORY)
+  @join__type(graph: PRODUCTS)
+  @join__type(graph: REVIEWS) {
+  me: User @join__field(graph: ACCOUNTS)
+  recommendedProducts: [Product] @join__field(graph: ACCOUNTS)
+  topProducts(first: Int = 5): [Product] @join__field(graph: PRODUCTS)
+}
+
+type Review @join__type(graph: REVIEWS, key: "id") {
+  id: ID!
+  body: String
+  author: User @join__field(graph: REVIEWS, provides: "username")
+  product: Product
+}
+
+type User
+  @join__type(graph: ACCOUNTS, key: "id")
+  @join__type(graph: REVIEWS, key: "id") {
+  id: ID!
+  name: String @join__field(graph: ACCOUNTS)
+  username: String
+    @join__field(graph: ACCOUNTS)
+    @join__field(graph: REVIEWS, external: true)
+  reviews: [Review] @join__field(graph: REVIEWS)
+}
+```
+
+</ExpansionPanel>
+
+## 3. Create and edit YAML configuration file
+
+The router's many features are configurable via a YAML configuration file. You set your options declaratively in YAML, then point your router to it at startup.
+
+Let's customize a common setting: the router's _supergraph listen address_. It's the network address and port on which the router receives client requests. By default the address is `127.0.0.1:4000`. As an exercise, let's change the port to `5555`.
+
+1. In your same working directory, create a file named `router.yaml`. Open it for editing.
+1. Add the following configuration that sets `supergraph.listen` to `127.0.0.1:5555`:
+
+  ```yaml title="router.yaml"
+  supergraph:
+    listen: 127.0.0.1:5555
+  ```
+
+## 4. Run the router
+
+When you're developing with the router locally, it's useful to run the router in _dev mode_. Dev mode is a collection of settings that make router development and debugging easier.
+
+Let's run the router in dev mode, using both the supergraph schema and YAML configuration files you created: 
+
+1. Run the router with these command-line options:
+    - `--dev` enables dev mode
+    - `--config` provides the path to your YAML configuration file
+    - `--supergraph` provides the path to your supergraph schema
```

Comment:
Is running with `--dev` now preferable to running `rover dev`, out of curiosity?

Can we briefly describe what running with `--dev` will do precisely, in one of those expandable drawers?

Reply:
add expansion panel